net/openssh/Portfile

```diff
@@ -50,9 +50,7 @@ if {${name} eq ${subport}} {
                         pam.patch \
                         patch-sandbox-darwin.c-apple-sandbox-named-external.diff \
                         patch-sshd.c-apple-sandbox-named-external.diff \
-                        0002-Apple-keychain-integration-other-changes.patch \
-                        macports-config.patch \
-                        patch-openbsd_compat-memmem-bug.diff
+                        macports-config.patch
 
     # We need a couple of patches
```

Comment:
This comment needs adjusting – it still references `0002-Apple-keychain-integration-other-changes.patch` that is now gone.